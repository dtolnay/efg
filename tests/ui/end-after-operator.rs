use efg::efg;

#[efg(unix ||)]
struct S;

#[efg((unix ||) && window)]
struct S;

fn main() {}
