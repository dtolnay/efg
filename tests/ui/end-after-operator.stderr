error: unexpected end of input
 --> tests/ui/end-after-operator.rs:3:1
  |
3 | #[efg(unix ||)]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `efg` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected an identifier
 --> tests/ui/end-after-operator.rs:6:15
  |
6 | #[efg((unix ||) && window)]
  |               ^
